README.md

```diff
@@ -40,7 +40,6 @@ This repository is broken into multiple components, each one implemented as a Un
 ```
     .
     ├── com.unity.netcode.gameobjects           # The core netcode SDK unity package (source + tests)
-    ├── com.unity.netcode.adapter.utp           # Transport wrapper for com.unity.transport
```

Comment:
this `README.md` file probably needs even more updates, but let's leave that for later.